client/src/components/UserProfileMyFlights.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <!-- TODO: Add filter spinner when loading -->
+  <!-- TODO: How many flights shall be shown -->
```

Comment:
All of the current year, just for a "quick" overview.
All flights of an user can be retrieved via FlightsAll and a filter.

Reply:
Current year or current season🤪

Reply:
todo added